tests/queries/0_stateless/02344_show_caches.sql

```diff
@@ -0,0 +1,2 @@
+-- Tags: no-fasttest
```

Comment:
The test runs in fractions of a millisecond, no reason to mark it "no-fasttest". Same for "02344_describe_cache". (just saying ... it has been like that before of course)

Reply:
No, actually there is a reason - there are no caches in fast test, so in fast test there will be a different .reference file

Reply:
Interesting, didn't know that.